external/dockerfile_functions.sh

```diff
@@ -611,11 +611,11 @@ generate_dockerfile() {
     base_docker_registry_dir="$9"
     check_external_custom_test=$10
```

Comment:
There are three more `check_external_custom_test` in this file and two more in common_functions.sh. Do they need to update to `check_external_custom`?

Reply:
I don't think so.. check_external_custom_test contained a path, whereas check_external_custom contains only binary information whether this path is present (or so is my understanding). I just updated the cases where they are used to test for 0 or 1... The better question would be whether these are ever used.. I can't find a real usage or passing of these variables inside either aqa-tests, nor TKG, they are just assigned, once propagated and then they just die out.. WDYT?

Reply:
They should be same thing as binary information.

Reply:
@sophia-guo okay, this means that the shell script had trouble picking up the 10th argument, and instead picked up the 1st one and appended "0" string.. ${10} solved it..
commit:
https://github.com/adoptium/aqa-tests/commit/73ea24fe236120cfa801c4b7c834f9ed8b1d4fda
grinder
https://ci.adoptium.net/view/Test_grinder/job/Grinder_ext_custom/38/consoleFull